[1.21.1] Added TPS activate event, modified BuildCameraTransform

Yesssssman opened the pull request:
Same goes with #2276, but in 1.21.1

Yesssssman committed:
Adapted changes mentioned in #2276 to 1.21.1

EchoEllet commented on src/main/java/yesman/epicfight/api/client/event/types/BuildCameraTransform.java:
```diff
@@ -23,9 +24,24 @@ public float getPartialTick() {
+        /// This means, you have modified the camera transform through this event
+        /// and don't want it to be overwrite by vanilla camera setups.
+        ///
+        /// @see MixinCamera#epicfight$setup
```
```suggestion
		/// Set to `true` to disable the vanilla camera setup process.
		/// Indicates that the camera transform has been modified through this event
		/// and should not be overwritten by vanilla camera setups.
		///
		/// @see MixinCamera#epicfight$setup
```

From [this comment](https://github.com/Epic-Fight/epicfight/pull/2276#discussion_r2580242713)

Yesssssman committed:
Update src/main/java/yesman/epicfight/api/client/event/types/BuildCameraTransform.java

Co-authored-by: Ellet <echo.ellet@gmail.com>

EchoEllet commented on src/main/java/yesman/epicfight/api/client/event/types/BuildCameraTransform.java:
```diff
@@ -23,8 +24,23 @@ public float getPartialTick() {
+		///
+		/// @see MixinCamera#epicfight$setup
+        public void setCancelVanillaCameraSetup(boolean flag) {
+            this.cancelVanillaCameraSetup = flag;
+        }
+
+        public boolean cancelVanillaCameraSetup() {
+            return this.cancelVanillaCameraSetup;
```
Same in [this comment](https://github.com/Epic-Fight/epicfight/pull/2276#discussion_r2580421692).

`cancelVanillaCameraSetup` has no side effects; it's just a pure immutable method (AKA getter).

EchoEllet commented on src/main/java/yesman/epicfight/api/client/event/types/BuildCameraTransform.java:
```diff
@@ -23,8 +24,23 @@ public float getPartialTick() {
             super(cameraApi, camera, partialTick);
+        }
+
+		/// Set to `true` to disable the vanilla camera setup process.
+		/// Indicates that the camera transform has been modified through this event
+		/// and should not be overwritten by vanilla camera setups.
+		///
+		/// @see MixinCamera#epicfight$setup
+        public void setCancelVanillaCameraSetup(boolean flag) {
+            this.cancelVanillaCameraSetup = flag;
+        }
```
Does `cancelVanillaCameraSetup` do anything? I don't see any usage in `EpicFightCameraAPI` or other classes.

Is there a reflection or dynamic method loading?

Yesssssman committed:
Fixed BuildCameraTransform.Pre

Fixed BuildCameraTransform.Pre not to cancel the vanilla setup process if it didn't call Camera#setPosition or Camera#setRotation.

Yesssssman committed:
simple method namefix
